run_configs/jules/build_jules.py

```diff
@@ -13,26 +13,37 @@
 from fab.steps.compile_fortran import compile_fortran
 from fab.steps.find_source_files import find_source_files, Exclude
 from fab.steps.grab.fcm import fcm_export
-from fab.steps.grab.prebuild import grab_pre_build
 from fab.steps.link import link_exe
 from fab.steps.preprocess import preprocess_fortran
 from fab.steps.root_inc_files import root_inc_files
+from fab.tools import Ifort, Linker, ToolBox
 
 logger = logging.getLogger('fab')
 
 
+class MpiIfort(Ifort):
+    '''A small wrapper to make mpif90 available.'''
+    def __init__(self):
+        super().__init__(name="mpif90", exec_name="mpif90")
+
+
```

Comment:
Indeed, that is my current thinking, but I am getting second thoughts ... I think it might be better if we used a Decorator pattern, that might make it much easier to handle compiler-specific flags (which after all also needs to be used in linking as well, e.g. omp flags etc).

But I think this will just change the way we implement compiler-wrapper (mpif90 around a standard compiler) and linker (which I think should pretty much be a compiler with the specified compiler flags, but additionally adds linker-flags for libs etc). It should not affect anything implemented in this PR, and we will verify what is best once we start working on compiler flags.